librz/core/cmd/cmd_print.c

```diff
@@ -5418,399 +5188,6 @@ RZ_IPI int rz_cmd_print(void *data, const char *input) {
 			break;
 		}
 		goto beach;
-	case 'D': // "pD"
-	case 'd': // "pd"
-	{
-		ut64 use_blocksize = core->blocksize;
-		ut8 bw_disassemble = false;
-		ut32 pd_result = false, processed_cmd = false;
-		bool formatted_json = false;
-		if (input[1] && input[2]) {
-			// "pd--" // context disasm
-			if (!strncmp(input + 1, "--", 2)) {
-				char *offs = rz_str_newf("%s", input + 2);
-				if (offs) {
-					ut64 sz = rz_num_math(core->num, offs);
-					char *fmt;
-					if (((st64)sz * -1) > core->offset) {
-						// the offset is smaller than the negative value
-						// so only print -offset
-						fmt = rz_str_newf("d %" PFMT64d, -1 * core->offset);
-					} else {
-						fmt = rz_str_newf("d %s", input + 2);
-					}
-					if (fmt) {
-						rz_cmd_print(core, fmt);
-						strcpy(fmt + 2, input + 3);
-						rz_cmd_print(core, fmt);
-						free(fmt);
-					}
-					free(offs);
-				}
-				ret = 0;
-				goto beach;
-			}
-		}
-
-		const char *sp = NULL;
-		if (input[1] == '.') {
-			sp = input + 2;
-		} else {
-			sp = strchr(input + 1, ' ');
-		}
-		if (!sp && (input[1] == '-' || IS_DIGIT(input[1]))) {
-			sp = input + 1;
-		}
-		if (sp) {
-			int n = (int)rz_num_math(core->num, rz_str_trim_head_ro(sp));
-			if (!n) {
-				goto beach;
-			}
-			use_blocksize = n;
-		}
-
-		if (core->blocksize_max < use_blocksize && (int)use_blocksize < -core->blocksize_max) {
-			eprintf("This block size is too big (%" PFMT64u "<%" PFMT64u "). Did you mean 'p%c @ 0x%08" PFMT64x "' instead?\n",
-				(ut64)core->blocksize_max, (ut64)use_blocksize, input[0], (ut64)use_blocksize);
-			goto beach;
-		} else if (core->blocksize_max < use_blocksize && (int)use_blocksize > -(int)core->blocksize_max) {
-			bw_disassemble = true;
-			l = use_blocksize; // negative
-			use_blocksize = -use_blocksize;
-		} else {
-			l = use_blocksize;
-		}
-		// may be unnecessary, fixes 'pd 1;pdj 100;pd 1' bug
-		rz_core_block_read(core);
-
-		switch (input[1]) {
```

Comment:
where are all these following commands going?

Reply:
They were ported to the rzshell https://github.com/rizinorg/rizin/pull/2552